### DISCUSSION TITLE
Changing the encryption key size

### DISCUSSION THREAD

**Discussion by EdvinLndh** [eProsima/Fast-DDS#4711] [Q&A] [answered]
The documentation states that AES-GCM supports both 128 and 256 bit key sizes. How do I configure which key size is used?

**Comment by MiguelCompany** [comment, marked as answer]
Setting property `dds.sec.crypto.keysize` on both reader and writer is what you need to do.
Using value `128` will use 128-bit keys. Any other value will use 256-bit keys.

I've looked at the [corresponding section](https://fast-dds.docs.eprosima.com/en/latest/fastdds/property_policies/security.html#cryptographic-plugin-settings) of the documentation, and I see that property is not documented. We will soon fix the documentation.
